test-site/src/components/MapPin.tsx

```diff
@@ -0,0 +1,20 @@
+import { PinComponent } from "@yext/search-ui-react";
+import { Location } from '../pages/LocationsPage'
+
+export const MapPin: PinComponent<Location> = () => {
```

Comment:
do we need this export if we already export the MapPin as the default below?

Reply:
removed the default export